introduction.md

```diff
@@ -6,33 +6,31 @@ layout: documentation
 
 # About openHAB
 
-openHAB is **software** for integrating different home automation systems and technologies into one single solution that allows over-arching automation rules, and that offers uniform user interfaces.
+openHAB is **software** for integrating different home automation systems, devices and technologies into one single solution.
+It provides uniform user interfaces, and a common approach to automation rules across the entire system, regardless of the number of manufacturers and sub-systems involved.
 
-But openHAB is also a **community** of users, contributors and maintainers.
+openHAB is also a **community** of users, contributors and maintainers, working together on an open-source, interoperable approach to home automation.
 
 ## openHAB Structure
 
-openHAB 2 is an open-source solution based on the [Eclipse SmartHome](https://eclipse.org/smarthome/) framework. It is written in Java and uses [Apache Karaf](http://karaf.apache.org/) together with [Eclipse Equinox](https://www.eclipse.org/equinox/) as an OSGi runtime and bundles this with [Jetty](https://www.eclipse.org/jetty/) as an HTTP server.
+openHAB 2 is based on the [Eclipse SmartHome](https://eclipse.org/smarthome/) framework, and is written in Java.
+It uses [Apache Karaf](http://karaf.apache.org/) together with [Eclipse Equinox](https://www.eclipse.org/equinox/) to create an Open Services Gateway initiative (OSGi) runtime environment.
+[Jetty](https://www.eclipse.org/jetty/) is used as an HTTP server.
 
-openHAB is highly modular software, which means that the base installation (the "runtime") can be extended through different kinds of "add-ons", either to communicate with new home automation solution, or to offer a new kind of user interface.
+openHAB is highly modular software, and can be extended through "add-ons".
+Add-ons give openHAB a wide array of capabilities, from User Interfaces, to the ability to interact with a large and growing number of physical Things.
```

Comment:
Maybe consider turning this into one nested sentence or change the second one a bit.
![image](https://user-images.githubusercontent.com/7161177/32692047-7b7bb78a-c711-11e7-88f8-92545de82c77.png)

Reply:
I respectfully disagree.  One of the things I have done generally is to eliminate long sentences.  I have found that people who do not have English as a first language, and who are not very fluent have real problems parsing long, complex sentences.  My assumption is that many readers of the documentation are not native English speakers.

In this case, having the first sentence finish by introducing the name Add-ons, and then having the next sentence start with Add-ons makes sense.  The subject of the first sentence is openHAB.  The subject of the second sentence is Add-ons.

My preference would be to keep it as-is.  What do you say?

Reply:
I agree with @bgilmer77.
Apart from that the ", and" sounds weird. How about "which" or "that":

> openHAB is a highly modular application which can be extended through various "Add-ons".

Reply:
Agree - openHAB is highly modular software that can be..."  I will make this change.

@ThomDietrich, if you are still around, could you please come up on gitter?  I have a question about linter I want to ask you

Reply:
This comment has been addressed.